integration_test/scripts/publish-images.sh

```diff
@@ -1,8 +1,14 @@
 #!/bin/bash
 # imageProject publishFlag repositoryString
```

Comment:
please update this list for the final PR

Reply:
done